src/assets/scss/application.scss

```diff
@@ -290,4 +290,59 @@ $nav-active-border-thickness: 5px;
   display: block;
   margin: govuk-spacing(2) 0;
   @include govuk-font($size: 48, $weight: bold);
-}
\ No newline at end of file
+}
+
+.language-select {
+  margin: govuk-spacing(2) 0;
+
+  &::after {
+    content: "";
+    display: block;
+    clear: both;
+  }
+
+  .language-select__list {
+    border-bottom: 1px solid $govuk-border-colour;
+    margin-bottom: 0;
+    float: right;
+    text-align: right;
+    @include govuk-media-query($until: tablet) {
+      float: none;
+      text-align: left;
+    }
+  }
+
+  .language-select__link {
+    @include govuk-font($size: 16);
+    padding: 0.3125em;
+
+    &:visited {
```

Comment:
Seeing as the `govuk-link--no-visited-state` modifier class has been used on the link element, is this rule necessary and why?

Reply:
removed